src/components/button-link/button-link.astro

```diff
@@ -0,0 +1,32 @@
+---
+import type React from 'react';
+
+export interface Props {
+  url?: string;
+  children: React.ReactNode;
+  disabled?: boolean;
+  className?: string;
+  secondary?: boolean;
+  isExternal?: boolean;
+}
+
+const { url, className = "", secondary = false, disabled = false, isExternal} = Astro.props;
+const resolvedIsExternal = isExternal !== undefined ? isExternal : url?.startsWith("http");
+---
+
+<a
+  class={`font-bold text-lg px-4 py-4 bg-button rounded-[60px] inline-block leading-4 hover:bg-button-hover not-prose
+    ${secondary ? "bg-primary text-white hover:bg-primary-hover" : "text-text-inverted"}
+    ${secondary ? "text-text" : ""}
+    ${className || ""}
+    ${disabled ? "opacity-50 pointer-events-none" : ""}`}
```

Comment:
I am a bit concerned that the contrast is too low. I think it might be better to use the secondary button styling for the disabled, e.g. changing the line 19 to `${secondary || disabled ? "bg-primary text-white hover:bg-primary-hover" : "text-text-inverted"}`? I think a greyed out button looks better too (and hopefully, the secondary colour will always be a fairly dark one in the future). WDYT?